src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -461,7 +461,7 @@ void unpack_i4f16(const ov::SoPtr<ov::ITensor>& from,
         unpackedToI8[unpackedIdx++] = upc(lo4(*(pSrc)));
         unpackedToI8[unpackedIdx++] = upc(hi4(*(pSrc)));
         if (unpackedIdx == VECSIZE) {
-            __m128i i8vec = _mm_loadu_si64(reinterpret_cast<__m128i*>(unpackedToI8));
+            __m128i i8vec = _mm_loadl_epi64(reinterpret_cast<__m128i*>(unpackedToI8));
```

Comment:
would it make sense to have both but under ifdef? like

```
#if defined(__GNUC__) && __GNUC__ < 9
```

Reply:
@dmatveev i dont think it worth, since we are using loadl_epi64 everywhere in the code, so only here when corner case handling was written with different intrinsics.